File: src/main/java/com/core/base/corebase/repository/ReviewMemberRepository.java
package com.core.base.corebase.repository;

import com.core.base.corebase.domain.review.ReviewMember;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;
import java.util.Optional;

public interface ReviewMemberRepository extends JpaRepository<ReviewMember, Long> {
    List<ReviewMember> findByReviewId(Long reviewId);
    Optional<ReviewMember> findByReviewIdAndMemberId(Long reviewId, Long memberId);
}

File: src/main/kotlin/com/core/base/corebase/repository/ReviewMemberRepository.kt
package com.core.base.corebase.repository

import com.core.base.corebase.domain.review.ReviewMember
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface ReviewMemberRepository : JpaRepository<ReviewMember, Long> {
    fun findByReviewId(reviewId: Long): List<ReviewMember>
    fun findByReviewIdAndMemberId(reviewId: Long, memberId: Long): ReviewMember?
}